doc/api.md

```diff
@@ -742,6 +744,60 @@ Unreserves a path for a publishing application. Returns success or failure only.
 - If a path reservation exists for the supplied base_path but a different
   publishing_app, the command will fail.
 
+## `PUT /publish-intent(/*base_path)`
+
+[Request/Response detail][put-intent-pact]
+
+Used to create or update a publishing intent for a particular base_path and
+reserve the base path in the publishing-api. A publishing intent sets when the TTL
+cache should be cleared.
+
+### Path parameters
+
+- `base_path`
+  - Identifies the path to create the publishing intent for
+
+### JSON parameters:
+
+- `publishing_app` *(required)*
+  - The name of the application making this request, words separated with
+    hyphens.
+- `rendering_app` *(required)*
+  - The name of the application rendering the content the base path refers to,
+    words separated with hyphens.
+- `publish_time` *(required)*
+  - The `DateTime` the content is set to be published.
+- `routes` *(optional)*
+  - `Array` - Fields supported are path and type. `path` can either be an exact
+    base_path or a subpath. If `path` is an exact base_path then `type` must be
+    `exact`, and if `path` is a subpath then `type` must have value `prefix`.
+    Required if you need a route to be added to the router-api for the given
+    base_path.
+
+### State changes
+
+- `base_path` is reserved for use of the given `publishing_app`.
+- Creates `PublishingIntent` in content-store to record when TTL cache should
+  expire.
+- If `routes` are given, adds a route set in router-api.
+
+## `DELETE /publish-intent(/*base_path)`
+
+Used to delete the publishing intent for a particular base_path. A publishing
+intent sets when the TTL cache should be cleared.
+
+[Request/Response detail][delete-intent-pact]
+
+### Path parameters
+
+- `base_path`
+  - Identifies the path to delete the publishing intent for
+
+### State changes
+
+- The `PublishingIntent` for the passed in base_path is removed from the
```

Comment:
Does this have any affect on the router?

Reply:
Nope not that i'm aware of!